### PR TITLE
[12.0][MIG] account_partner_reconcile

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: account_partner_reconcile/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The button is visible only to users that belong to the accounting groups
+"Accountant" or "Adviser".
```
Comment (luismontalba) [inline account_partner_reconcile/readme/CONFIGURE.rst:2]: "Billing" or "Billing Manager". To carry out the matching, users need to have activated the check "Show Full Accounting Features".

**Review by luismontalba — APPROVED** [review]
Tested on runbot, working fine, minor changes proposed in README

File: account_partner_reconcile/views/res_partner_view.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_partner_view_buttons" model="ir.ui.view">
```
Comment (tbaden) [inline account_partner_reconcile/views/res_partner_view.xml:4]: cosmetical change: use original id (not relevant for approvement)
> prefer naming the inherited view after the original over using a name which follows the guidelines.

```
<record id="original_id" model="ir.ui.view">
    <field name="inherit_id" ref="original_module.original_id"/>
```

**Review by tbaden — APPROVED** [review]

**Comment by emagdalenaC2i** [comment]
@pedrobaeza Please. take a look at https://github.com/OCA/account-reconcile/tree/12.0/account_partner_reconcile

I think this module is already migrated and merged in /OCA/account-reconcile

**Comment by emagdalenaC2i** [comment]
@mreficent Please, confirm if this PR is the same as the one in /OCA/account-reconcile
https://github.com/OCA/account-reconcile/tree/12.0/account_partner_reconcile

As you can see in #201 is moved and already merged

**Comment by pedrobaeza** [comment]
Right, I have removed it from this repo in 7da0e3d14976ba03747ef9bfcb2faf84e3385b92

**Comment by MiquelRForgeFlow** [comment]
Oh my... :S

**Pull request merged**